projects/cashmere/src/lib/tabs/tab-set/tab-set.component.ts

```diff
@@ -405,6 +423,14 @@ export class TabSetComponent implements AfterContentInit {
         this._stopTabSubscriptionSubject.next();
         this._tabs.forEach(t => t.tabClick.pipe(takeUntil(this._stopTabSubscriptionSubject)).subscribe(() => this._setActive(t)));
 
+        // If using tabs with routing, listen for changes to the routerLinkActive state
+        if ( this._routerEnabled ) {
+            this._tabs.forEach(t => t._routerActiveChange.pipe(takeUntil(this._stopTabSubscriptionSubject)).subscribe(tab => {
```

Comment:
Does `_routerActiveChange` only fire when the tab becomes active? Or does it also fire when the tab becomes inactive?

Reply:
Never mind, I can see in the next file that it only fires when status is `true`. Carry on.